src/server/ua_server_config.c

```diff
@@ -28,6 +28,10 @@ UA_ServerConfig_clean(UA_ServerConfig *config) {
 
     /* Custom DataTypes */
     /* nothing to do */
+    for (size_t i = 0; i < config->connectionManagersSize; i++) {
+        UA_ConnectionManager* cm = config->connectionManagers[i];
+        UA_free(cm->initialConnectionContext);
```

Comment:
Use `el->free`.
https://github.com/open62541/open62541/blob/5bac022548b0ff8167278a042a75c4fcfbdcc47d/include/open62541/plugin/eventloop.h#L98

Reply:
Does the el->free also free the initialConnectionContext?
And if it does we should maybe overthink that, because it is the responsibility of the caller to allocate memory and assign the initial connection context on a connection manager. The caller may even assign a statically allocated address in the program-text segment which should not be freed at all. Or am I missing something?

Reply:
You are right.
The initialContext is not touched.
Only the EventSource itself is freed.